website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,114 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as running queries, loading or exporting data, and creating tables. While not necessary for basic operations, these options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Select your BigQuery project from the left-hand menu
```

Comment:
where do they start from ? from the user perspective, this isn't that clear and i wouldn't know where to go? is it from account settings -> connections -> click on the bigquery? or is it account settings -> projects -> click on project -> and then dev connection -> edit?

i think we can make this much more clearer so its not so confusing?